cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -14,29 +14,29 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: heapster-v1.3.0-beta.0
+  name: heapster-v1.3.0-beta.1
   namespace: kube-system
   labels:
     k8s-app: heapster
     kubernetes.io/cluster-service: "true"
-    version: v1.3.0-beta.0
+    version: v1.3.0-beta.1
 spec:
   replicas: 1
   selector:
     matchLabels:
       k8s-app: heapster
-      version: v1.3.0-beta.0
+      version: v1.3.0-beta.1
   template:
     metadata:
       labels:
         k8s-app: heapster
-        version: v1.3.0-beta.0
+        version: v1.3.0-beta.1
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
         scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster:v1.3.0-beta.0
+        - image: gcr.io/google_containers/heapster:v1.3.0-beta.1
```

Comment:
Should be `gcr.io/google_containers/heapster-amd64:v1.3.0-beta.1`. The same in the other places.